src/Controller/Simulation.java

```diff
@@ -24,6 +21,7 @@
     private final House house;
     private boolean running;
 
+    private boolean isAway;
```

Comment:
maybe we should initialize isAway with a default value in the Simulation constructor?